compiler_opt/tools/extract_ir.py

```diff
@@ -76,6 +76,18 @@
     'passed in the local case.')
 flags.mark_flags_as_required(['output_dir'])
 
+flags.DEFINE_string(
+    'cmd_section_name', '.llvmcmd',
+    'Name of the section name passed to llvm-objcopy. For linux systems, the '
+    'default .llvmcmd is correct. On macos, one should use __LLVM,__cmdline'
```

Comment:
Technically this is not about the OS but about the file format (for example, you might be running training on macOS but using binaries compiled for Linux), so I think it's more accurate to say "ELF" and "Mach-O" rather than "Linux" and "macOS" respectively.

Reply:
fixed - thanks!